DataFormats/CSCDigi/src/CSCCorrelatedLCTDigi.cc

```diff
@@ -101,11 +101,11 @@ bool CSCCorrelatedLCTDigi::getEightStrip() const { return (strip >> kEightStripS
 /// return the fractional strip
 float CSCCorrelatedLCTDigi::getFractionalStrip(const uint16_t n) const {
   if (n == 8) {
-    return 0.125f * (getStrip() + 1) - 0.0625f;
+    return 0.125f * (getStrip(n) + 1) - 0.0625f;
```

Comment:
Is this also a bug fix?

Reply:
`getFractionalStrip()` with n==4 or n==8 is not being used yet in the code, so changing the definition would not have an effect on the unit tests.